resources/views/users/reserved/join_users.blade.php

```diff
@@ -33,38 +33,25 @@
                 {{-- Modal --}}
                 <div class="modal fade" id="other-member{{ $user->id }}">
                     <div class="modal-dialog">
-                        <div class="modal-content border-warning">
-                            <div class="modal-header">
-                                <div class="modal-title mx-auto">OTHER MEMBER</div>
-                            </div>
+                        <div class="modal-content border border-2 border-warning">
+
                             <div class="modal-body">
-                                <div class="avatar mx-auto">
-                                    @if($user->avatar)
-                                    <img src="{{ asset('storage/avatars/'.$user->avatar) }}" alt="" class="avatar-md">
+                                <div class="modal-title text-center fs-2 my-3 fw-bold">OTHER MEMBER</div>
+                                    <div class="avatar mx-auto">
+                                        @if($user->avatar)
+                                        <img src="{{ asset('storage/avatars/'.$user->avatar) }}" alt="" class="avatar-md">
+                                        @else
+                                        <i class="fa-solid fa-circle-user fa-2xl text-secondary text-center"></i>
+                                        @endif
+                                    </div>
+                                    {{-- show user's level icon --}}
+                                    <h3 class="mx-auto text-center" id="username">{{ $user->user_name }}</h3>
+                                    @if($user->comment)
```

Comment:
you can also check if the user->comment is null or blank by using == ' ' or Isblank function

Reply:
Thank you for your help.
I could not use the isBlank function, empty function worked correctly. This time I want to use @if, which we used in the previous lesson.